### DISCUSSION TITLE
Notice for Contributors

### DISCUSSION THREAD

**Discussion by KeironO** [AberystwythSystemsBiology/limbus#121] [General]
Hey everyone,

As I transition away from the project it's probably a good time to start enforcing some contributing rules.

For starters, all new contributions are required to follow a standard naming convention for branches:

- `feature/name-of-the-feature`
- `fix/name-of-the-fix`

Once we start getting used to this, I should be able to set up CD as the current deployment pipeline is too involved.

Also, any large PRs (+2,000loc changes) will likely be rejected. I understand that sometimes this is unavoidable (and leniency will be applied) - but if a PR can be easily broken up into several smaller PRs then it needs to be in several PRs.

Let's try and keep stuff tight going forward, I had to review 17,000 loc changes last night and it's not really doable long term.

If you're an internal contributor, I will try to review your code within 24 hours. If you're external then it's going to take 72 hours.

Sorry about this.

Best wishes,

Keiron

**Comment by reynamonal3** [comment]
Best wish to all city tht contribute to the community
